app/views/pages/get-started/contact-us-tutorial/building-the-ui.liquid

```diff
@@ -0,0 +1,99 @@
+---
+metadata:
+  title: Building the User Interface
+  description: Follow these step-by-step tutorials to build a Contact Us form on platformOS
+converter: markdown
+---
+
+To implement the _Contact Us_ form, we'll create a homepage with two fields: `email` and `body`, where users can enter their contact information. 
+
+{% include 'alert/tip', content: 'Creating a homepage is a bit special compared to creating a regular [page](/developer-guide/pages/pages). For detailed documentation, refer to the [Homepage](/developer-guide/pages/pages#homepage) documentation.' %}
+
+
+## Requirements
+
+* Followed all steps of the [Quick Setup Guide](/get-started/contact-us-tutorial/#quick-setup-guide) for building 
+* HTML knowledge
+* [Pages](/developer-guide/pages/pages)
+
+## Create a directory for pages
+
+First, create a directory for your pages within the `app` directory:
```

Comment:
Ok, now I get it. You did not wanted to use `pos-cli init` to get the directory structure for the project and instead we are creating everything manually. Was that intentional? Isn't `init` exactly so it all would be faster?

Reply:
this was intentional because we don't want to teach people how to use platformOS most efficiently, but to explain how it works :)

Reply:
Exactly! :)